railties/lib/rails/commands/test.rb

```diff
@@ -1,9 +1,9 @@
 require "rails/test_unit/minitest_plugin"
 
 if defined?(ENGINE_ROOT)
-  $: << File.expand_path("test", ENGINE_ROOT)
+  $LOAD_PATH << File.expand_path("test", ENGINE_ROOT)
 else
-  $: << File.expand_path("../../test", APP_PATH)
+  $LOAD_PATH << File.expand_path("../../test", APP_PATH)
 end
 
-exit Minitest.run(ARGV)
+Minitest.autorun
```

Comment:
I think we should `require minitest/autorun` here, so it will fix the issue where there is a double run.